deploy.mk

```diff
@@ -0,0 +1,9 @@
+install:
+	helm upgrade monitor ../charts/monitor --install --namespace deis --set grafana.org=${IMAGE_PREFIX},grafana.docker_tag=${VERSION},telegraf.org=${IMAGE_PREFIX},telegraf.docker_tag=${VERSION}
```

Comment:
we are still using our in-house `influxdb` sub-component/chart, right?  If so, we'd want to add it to the list here and below...